datasets/kalenderreferentie/kalenderreferentie.json

```diff
@@ -0,0 +1,299 @@
+{
+  "type": "dataset",
+  "id": "kalenderreferentie",
+  "title": "Kalenderreferentie",
+  "status": "beschikbaar",
+  "description": "Een generieke hiërarchisch dataset met datum (yyyy-mm-dd) als laagste granulariteitsniveau.",
+  "version": "0.0.1",
+  "crs": "EPSG:28992",
+  "theme": ["kalender", "referentie", "datum"],
+  "owner": "Gemeente Amsterdam",
+  "accrualPeriodicity": "dagelijks",
+  "publisher": "OIS",
+  "keywords": ["kalender", "referentie", "datum"],
+  "contactPoint": {
+    "name": "",
+    "email": "datapunt@amsterdam.nl"
+  },
+  "tables": [
+    {
+      "id": "datum",
+      "type": "table",
+      "auth": "FP/MDW",
```

Comment:
Waarom is deze dataset afgeschermd? Het lijkt me allemaal behoorlijk open data.

Reply:
Goeie, ik zal em weghalen